src/nanoemoji/config.py

```diff
@@ -87,6 +87,12 @@
     "Whether to fail or continue with a warning when picosvg cannot compute "
     "affine between paths that normalize the same.",
 )
+flags.DEFINE_integer(
+    "clipbox_quantization",
+    None,
+    "Whether to round COLR clip boxes to multiples of given integer.",
```

Comment:
Round toward negative/positive infinity hopefully, not the nearest multiple? - should describe so as well